docs/developer-docs/latest/developer-resources/database-apis-reference/graphql-api.md

```diff
@@ -0,0 +1,476 @@
+  documents {
+    data {
+      id
+      attributes {
+        title
+        locale
+        categories {
+          data {
+            id
+            attributes {
+                name
+            }
+          }
+        }
+      }
+    }
+    meta {
+      pagination {
+        page
+        pageSize
+        total
+        pageCount
+      }
+    }
+  }
+}
+
+```
+
+:::
+
+<!-- ? is this part 👇 still relevant? -->
+<!-- ### Fetch dynamic zone data
+
+Dynamic zones are union types in graphql so you need to use fragments to query the fields.
+
+:::request Query
+```graphql
+query {
+  restaurants {
+    dz {
+      __typename
+      ... on ComponentDefaultClosingperiod {
+        label
+      }
+    }
+  }
+}
+```
+::: -->
+
+## Mutations
+
+| Argument     | Type   | Description      |
+| ------------ | ------ | ---------------- |
+| `id`         | String | Entry id         |
+| `attributes` | Object | Entry attributes |
+
+### Create a new entry
+
+:::request Mutation
+
+```graphql
+mutation createArticle {
+  createArticle(data: { title: "Hello", relation: 1 }) {
+    data {
+      id
+      attributes {
+        title
+      }
+    }
+  }
+}
+```
+
+:::
+
+<!-- ? is it still relevant 👇 ? -->
+<!-- The implementation of the mutations also supports relational attributes. For example, you can create a new `User` and attach many `Restaurant` to it by writing your query like this:
+
+:::request Mutation
+
+```graphql
+mutation {
+  createUser(
+    input: {
+      data: {
+        username: "John"
+        email: "john@doe.com"
+        restaurants: ["5b51e3949db573a586ad22de", "5b5b26619b0820c1c2fb79c9"]
+      }
+    }
+  ) {
+    user {
+      username
+      email
+      restaurant {
+        name
+        description
+        price
+      }
+    }
+  }
+}
+```
+
+::: -->
+
+### Update an existing entry
+
+:::request Mutation
+
+```graphql
+mutation updateArticle {
+  updateArticle(id: "1", data: { title: "Hello", relation: 1 }) {
+    data {
+      id
+      attributes {
+        title
+      }
+    }
+  }
+}
+```
+
+:::
+
+<!-- ? is it still relevant? -->
+<!-- You can also update relational attributes by passing an ID or an array of IDs (depending on the relationship).
+
+:::request Mutation
+```graphql
+mutation {
+  updateRestaurant(input: {
+    where: {
+      id: "5b5b27f8164f75c29c728110"
+    },
+    data: {
+      chef: "5b51e3949db573a586ad22de" // User ID
+    }
+  }) {
+    restaurant {
+      chef {
+        username
+        email
+      }
+    }
+  }
+}
+```
+::: -->
+
+### Delete an entry
+
+:::request Mutation
+
+```graphql
+mutation deleteArticle {
+  deleteArticle(id: 1) {
+    data {
+      id
+      attributes {
+        title
+      }
+    }
+  }
+}
+```
+
+:::
+
+## Filters
+
+Queries can accept a `filters` parameter with the following syntax:
+
+`filters: { field: { operator: value } }`
+
+Logical operators (`and`, `or`) can also be used and accept arrays of objects.
+
+The following operators are available:
```

Comment:
The full list of supported operators is listed here.

https://github.com/strapi/strapi/blob/releases/v4/packages/plugins/graphql/server/services/builders/filters/operators/index.js#L5

I believe there are some operators to add/remove to/from your list